Fixes:
CMake Error at /usr/share/cmake-3.22/Modules/FindPkgConfig.cmake:890 (message):
  None of the required 'click-0.4' found
---
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7bf9aa1..ec3af2e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,7 +51,7 @@ pkg_search_module(ACCOUNTSSERVICE REQUIRED accountsservice)
 pkg_search_module(GEONAMES REQUIRED geonames)
 pkg_search_module(ICU REQUIRED icu-i18n)
 pkg_search_module(ANDR_PROP libandroid-properties)
-pkg_search_module(CLICK REQUIRED click-0.4)
+pkg_check_modules(CLICK REQUIRED click-0.4)
 pkg_check_modules(QTGSETTINGS REQUIRED gsettings-qt)
 
 find_program(XGETTEXT_BIN xgettext)
